packages/features/bookings/Booker/types.ts

```diff
@@ -30,19 +50,8 @@ export interface BookerProps {
    */
   eventMetaChildren?: React.ReactNode;
 
-  /**
-   * Whether is a team or org, we gather basic info from both
-   */
-  entity: {
-    fromRedirectOfNonOrgLink?: boolean;
-    considerUnpublished: boolean;
-    isUnpublished?: boolean;
-    orgSlug?: string | null;
-    teamSlug?: string | null;
-    name?: string | null;
-    logoUrl?: string | null;
-    eventTypeId?: number | null;
-  };
+  /** Entity configuration - orgSlug is auto-resolved from event data in Platform Booker */
```

Comment:
We can remove this comment - this component should not explain what is resolved how because that logic lives upper in hierarchy.